playground/src/Editor/db.ts

```diff
@@ -0,0 +1,27 @@
+const DB_URL = import.meta.env.PROD
+  ? "https://api.astral-1ad.workers.dev"
+  : "http://0.0.0.0:8787";
+
+/**
+ * Get a value from the database.
+ */
```

Comment:
This is a bit abstract (same as the `get/set` method names). I would be leaning towards more specific method names (given a playground id, restores the playground state) and for set, given the playground state, persists the state and returns the key.